RegularExpressionAndExceptionHandling/RegularExpressionAndExceptionHandling/ValidPhoneNumber.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+
+namespace RegularExpressionAndExceptionHandling
+{
+    /// <summary>
+    /// ValidPhoneNumber class uses a regular expression to validate all valid forms of United States phone numbers.
+    /// It passes in a string, phoneNumber, to validate against a pattern. 
```

Comment:
Just a reminder that XML comments should describe what a method does rather than how, preserving the ability to change the "how" over time will keeping the "what" the same for callers.